docs/content/2.deployment/2.cloudflare_pages.md

```diff
@@ -0,0 +1,89 @@
+# Deploying Elk to Cloudflare Pages
+
+## Prerequisites
+- A Cloudflare account is required. The _Free_ Tier has the necessary features to support Elk deployment to Cloudflare Pages. Once you have a Cloudflare account set up, you will need to:
+  - Create a Cloudflare KV Namespace. Make a note of the namespace `id`, which you'll need to configure your build.
+  - Generate a Cloudflare API token with read/write privileges to the KV Namespace you just created.
```

Comment:
It might be useful to extract the content on this in `1.netlify.md` into a single shared page, which can benefit from the screenshots, etc. Alternatively, we could share wording or link between the two pages?

Reply:
Extracted the shared content into `10.getting_started.md`